src/settings/target/target.rs

```diff
@@ -47,4 +46,28 @@ impl Target {
             }
         }
     }
+
+    pub fn routes(&self) -> Result<Vec<Route>, failure::Error> {
+        let mut routes = Vec::new();
+
+        if let Some(single_route) = &self.route {
+            if self.routes.is_some() {
```

Comment:
this case isn't covered in the else and should probably be brought out to the top - 

```rust
if &self.route.is_some() && &self.routes.is_some() {
  failure::bail!("You can specify EITHER `route` or `routes` in your wrangler.toml")
}
```

(also it shouldn't have a trailing semi-colon i think)

Reply:
what do you mean "this case isn't covered in the else"?

Reply:
in the `else if let Some(multi_route)` block, we do not check to see if there is also a `route` field. I see now that if they were both there, the first block would catch it, but I still find it a bit strange that we check if both exist after we check if one exists. I think I'd have it as its own separate block like I describe above even though they would be functionally equivalent.

Reply:
yeah covering it in both places would be redundant. i'll pull up the first case.